packages/design-system/src/components/ChoiceList/__snapshots__/ChoiceList.test.tsx.snap

```diff
@@ -5,29 +5,11 @@ exports[`ChoiceList Radio buttons and Checkboxes is a radio button group 1`] = `
   choices={
     Array [
       Object {
-        "disabled": undefined,
-        "inputClassName": "",
-        "inputRef": [Function],
-        "inversed": undefined,
         "label": "Choice 1",
-        "name": "spec-field",
-        "onBlur": undefined,
-        "onChange": undefined,
-        "size": undefined,
-        "type": "radio",
```

Comment:
Hm, would this snapshot test change be related to the `type` prop issue on typescript app like coverage tools frontend?